src/InitializeDatabaseScript.py

```diff
@@ -0,0 +1,201 @@
+import psycopg2
+import src.DBConfig as config
+import src.DebugLogFile as Dfl
+
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+
+
+class Database:
+    def __init__(self):
+        self.db_server_conn = None
+        self.database_params = None
+        self.db_cursor = None
+        self.db_name = None
+        self.bug_table_name = "Bugs"
+        self. debugLogFile = Dfl.debugLogFile()
+
+    # Open a connection to the host server that the database is stored on.
+    def connect(self):
+        try:
+            # Get the server params from Database.ini
+            params = config.config('postgres_server')
+            # Create a connection to the posgresql database server
+            self.db_server_conn = psycopg2.connect(**params)
+
+            self.db_cursor = self.db_server_conn.cursor()
+            # Print PostgreSQL Connection properties
+            print(self.db_server_conn.get_dsn_parameters(), "\n")
+
+            # Print PostgreSQL version
+            # possibly do not need class based cursor so this might need to be removed.
+            # TODO
+
+            self.db_cursor.execute("SELECT version();")
+            record = self.db_cursor.fetchone()
+            print("You are connected to - ", record, "\n")
```

Comment:
We should remove any and all print statement

Reply:
Removed